devguide/howtos/use-forwarded.socket.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Handling *.forwarded socket messages.
+layout: default
+---
+This page describes how to handle forwarded requests.
+------------------
+
+Some XAPI daemon uses a *.forwarded unix domain socket to receive "forwarded" requests from XAPI.
+It uses a different interface than XML-RPC over HTTP, and can be a bit confusing at first stance.
+
+Through this socket, XAPI forwards the request from a host using a JSON-serialized HTTP Request from the XAPI internal http library, it is defined as such :
```

Comment:
Please warp lines to facilitate review here. (Not a problem in this specific case but in general MD files should not exceed 80 characters per line.